docs/readthedocs/usersguide.md

```diff
@@ -174,25 +171,25 @@ An external simulation has to depend on `SimonaAPI <https://github.com/ie3-insti
 In order to run an external simulation, several requirements have to be fulfilled and a bunch of preparation steps have to be followed.
 
 .. note::
-    The external simulation is loaded via class loader. Thus, SIMONA and the external simulation should be built using the same JDK.
+The external simulation is loaded via class loader. Thus, SIMONA and the external simulation should be built using the same JDK.
+
+### Requirements
 
-Requirements
-------------
 
 **External simulation**
 
 - The external simulation should be implemented in its own project (repository).
 - The project should include the *shadowJar* gradle plugin (``id "com.github.johnrengelman.shadow" version "x.y.z"``).
-- A class (called *main class* here) needs to extend ``edu.ie3.simona.api.schedule.ExtSimulation`` and thus implement the two methods ``List<Long> initialize()´´ and ``List<Long> doActivity(long tick)``. The method ``initialize´´ is called when the external simulation needs to be initialized whereas the method ``doActivity´´ is called when time step ``tick`` is triggered. ``initialize´´ and ``doActivity`` must return a list of subsequent new ticks that the sub simulation should be scheduled at.
+- A class (called *main class* here) needs to extend ``edu.ie3.simona.api.schedule.ExtSimulation`` and thus implement the two methods ``List<Long> initialize()`` and ``List<Long> doActivity(long tick)``. The method ``initialize`` is called when the external simulation needs to be initialized whereas the method ``doActivity`` is called when time step ``tick`` is triggered. ``initialize`` and ``doActivity`` must return a list of subsequent new ticks that the sub simulation should be scheduled at.
 - For each data stream, a sub-interface of ``edu.ie3.simona.api.data.ExtDataSimulation`` needs to be implemented, such as ``edu.ie3.simona.api.data.ev.ExtEvSimulation``, and all methods of the interface have to be implemented. The *main class* could be the implementing class here.
-- In order for SIMONA to recognize the external simulation, a class ``edu.ie3.simona.api.ExtLink`` has to reside inside the project. This class has to extend ``edu.ie3.simona.api.ExtLinkInterface`` and implement the corresponding methods by returning the control stream and data stream implementations (could all be the same *main class*).
+- In order for SIMONA to recognize the external simulation, a class that extends ``edu.ie3.simona.api.ExtLinkInterface`` and a service file (``META-INF/service/edu.ie3.simona.api.ExtLinkInterface``) have to reside inside the project. The class has to implement the corresponding methods by returning the control stream and data stream implementations (could all be the same *main class*). The service file needs to contain the relative path to the class that extends ``edu.ie3.simona.api.ExtLinkInterface``.
```

Comment:
I like the new description of todos here, however:
- I think it could be clearer to the user if you split this point up into one point for the simulation class and one for the META-INF service file.
- Please include the full path for the META-INF directory, i.e. `src/main/resources/META-INF/services/edu.ie3.simona.api.ExtLinkInterface`
- You could also include a link to the [sample project](https://github.com/ie3-institute/ExtSimSample) (which needs adaptations to the new architecture as well btw).

Thanks!

Reply:
- Splited up the point for the simulation class and for the service file.
- Added a link to the sample project.